eclass/toolchain.eclass

```diff
@@ -577,7 +545,8 @@ do_gcc_PIE_patches() {
 	want_pie || return 0
 	use vanilla && return 0
 
-	tc_apply_patches "Applying pie patches ..." "${WORKDIR}"/piepatch/*.patch
+	einfo "Applying pie patches ..."
+	eapply "${WORKDIR}"/piepatch/*.patch
 
 	BRANDING_GCC_PKGVERSION="${BRANDING_GCC_PKGVERSION}, pie-${PIE_VER}"
 }
```

Comment:
Let's modernise the readarray bit below now we're req'ing EAPI 7+.

Reply:
bash-4.4 requires EAPI >= 8

Reply:
oops!